src/Akeneo/Tool/Bundle/MeasureBundle/Resources/public/context/router-context.ts

```
@@ -0,0 +1,11 @@
+import {createContext} from 'react';
+
+export interface RouterContextValue {
+  generate: (route: string, parameters?: {[param: string]: string}) => string;
+  redirect: (fragment: string, options?: object) => void;
+}
+
+export const RouterContext = createContext<RouterContextValue>({
+  generate: (route, parameters) => route + (parameters ? '?' + new URLSearchParams(parameters).toString() : ''),
+  redirect: () => undefined,
```

shouldn't we return an undefined here ? I dk, in this case I would throw an exception ? I don't know :/